database/clickhouse/README.md

```diff
@@ -1,13 +1,14 @@
 # ClickHouse
 
-`clickhouse://host:port?username=user&password=qwerty&database=clicks&x-multi-statement=true`
+`clickhouse://host:port?username=user&password=qwerty&database=clicks&x-multi-statement=true&x-table-engine=MergeTree`
 
 | URL Query  | Description |
 |------------|-------------|
 | `x-migrations-table`| Name of the migrations table |
+| `x-table-engine`| Table engine to use for the migrations table, defaults to TinyLog |
```

Comment:
rename to `x-migrations-table-engine` for clarity

Reply:
changed